addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/connector/RFXComTcpConnector.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2010-2014, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.rfxcom.internal.connector;
+
+import java.io.IOException;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * RFXCOM connector for TCP/IP communication.
+ * 
+ * @author Pauli Anttila - Initial contribution
+ */
+public class RFXComTcpConnector implements RFXComConnectorInterface {
```

Comment:
This class is nowhere used - is this something to be supported in future? As it is pretty empty as well, maybe better remove it for now.